packages/samples/mvp-dapp/src/type-extends.d.ts

```diff
@@ -14,5 +14,6 @@ declare module '@ckb-js/kuai-core' {
     lumosConfig?: Config | 'aggron4' | 'lina'
     redisPort?: number
     redisHost?: string
+    redisOpt?: object
```

Comment:
The type can be more specific and imported from https://github.com/redis/ioredis/blob/main/lib/redis/RedisOptions.ts#L184